components/App.js

```diff
@@ -1,59 +1,78 @@
-var App = React.createClass({
+const App = React.createClass({
 
     getInitialState() {
         return {
             loading: false,
             searchingText: '',
             gif: {}
-        };
+        }
     },
 
     handleSearch: function(searchingText) {
         this.setState({
             loading: true
         });
 
-        this.getGif(searchingText, function(gif) {
-            this.setState({
-                loading: false,
-                gif: gif,
-                searchingText: searchingText
-            });
-        }.bind(this));
+        this.getGif(searchingText)
```

Comment:
deklaracja metody `getInitialState` jest zapisana w konwencji "skróconej" natomiast pozostałe metody w klasie są zdefiniowane przy użyciu:
```
nazwaMetody: function() {
}
```
konwencja skrócona to:
```
nazwaMetody() {
}
```

możesz to ujednolicić :)